go.mod

```diff
@@ -1,38 +1,36 @@
 module github.com/osmosis-labs/osmosis/v21
 
-go 1.20
+go 1.21
```

Comment:
AFAIK, there is no state-break between v1.20 and 1.21. Could someone please confirm?

I was getting warnings so updated this.